stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputViewStyle.kt

```diff
@@ -123,8 +141,37 @@ public data class MessageInputViewStyle(
     @ColorInt public val commandButtonRippleColor: Int?,
     @ColorInt public val attachmentButtonRippleColor: Int?,
     public val messageInputInputType: Int,
+    // Message reply customization, by default belongs to center content as well
+    @ColorInt public val messageReplyBackgroundColor: Int,
```

Comment:
Only a single option here because https://github.com/GetStream/stream-chat-android/pull/4679/files#diff-225a4dc6c1f949cc2e4033671afe994b7d1e7eebfe70ace1117d64524ac2d66fL144-R144 makes it so that only one option will ever be exercised since the replies inside the input/ composer are always owned by the currently logged-in user.